### ISSUE TITLE
Mobile menu x button missing focus 

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#16]
Language: English

Tool(s): visual check

Browser/Screen reader: IE and Firefox

WCAG 2.0 reference: 2.4.7 Focus visible

Details: When the page is magnified to 200% the user is tabbing through the menu but the close 'X' does not have a visible focus (other than a slight change in colour).
When the "X" is in focus the contrast is
Foreground:#555555
Background:#FFFFFF

The contrast ratio is: 7.5:1, but it does not have the focus outlining.

![image](https://user-images.githubusercontent.com/41083520/74243688-3080f680-4cae-11ea-9710-c49394d11317.png)

This is without focus - the colour contrast is lower
Foreground:#909090
Background:#FFFFFF

The contrast ratio is: 3.2:1

![image](https://user-images.githubusercontent.com/41083520/74243725-3e367c00-4cae-11ea-82ca-518e12b96a91.png)

**Comment by stef-barburou** [comment]
fixed in previous commit
